<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWidget</class>
 <widget class="QWidget" name="SearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="andOR">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Toggle AND/OR </string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Keyword</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="keyword_1"/>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="andORlabel_1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>AND</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Columns</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="columnSelect_1">
     <item>
      <property name="text">
       <string>ALL</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Keyword</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="keyword_2"/>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="andORlabel_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>AND</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Columns</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QComboBox" name="columnSelect_2">
     <item>
      <property name="text">
       <string>ALL</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Keyword</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="keyword_3"/>
   </item>
   <item row="7" column="3">
    <widget class="QLabel" name="andORlabel_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>AND</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Columns</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QComboBox" name="columnSelect_3">
     <item>
      <property name="text">
       <string>ALL</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="9" column="0" colspan="4">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Keyword</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QLineEdit" name="keyword_4"/>
   </item>
   <item row="10" column="3">
    <widget class="QLabel" name="andORlabel_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>AND</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Columns</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QComboBox" name="columnSelect_4">
     <item>
      <property name="text">
       <string>ALL</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_6">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Expectes a text file with the list of contig names sparated by comma, tab or space, presented in a single line or over multiple lines&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>ORF List File</string>
     </property>
    </widget>
   </item>
   <item row="12" column="3">
    <widget class="QLabel" name="andORlabel_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>AND</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_7">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Expectes a text file with the list of contig names sparated by comma, tab or space, presented in a single line or over multiple lines&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Contig List File</string>
     </property>
    </widget>
   </item>
   <item row="13" column="3">
    <widget class="QLabel" name="andORlabel_6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>AND</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="2">
    <widget class="QCheckBox" name="caseSensitive">
     <property name="text">
      <string>Case sensitive</string>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="QDialogButtonBox" name="okAndCancel">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="orfList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Expectes a text file with the list of contig names sparated by comma, tab or space, presented in a single line or over multiple lines&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="orfListButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="13" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="contigList">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Expectes a text file with the list of contig names sparated by comma, tab or space, presented in a single line or over multiple lines&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="contigListButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
